src/zoid/button/component.jsx

```diff
@@ -205,6 +206,37 @@ export const getAuthButtonComponent = memoize(() : ZoidComponent<ButtonProps> =>
                         action: 'auth'
                     };
                 }
+            },
+
+            scopes: {
+                type:          'array',
+                required:      true,
+                queryParam:    true,
+                validate:      ({ value }) => {
+                    return validateScopes(value);
+                },
+                queryValue:    ({ value }) => {
+                    return value.join(' ');
```

Comment:
Can we do comma separated? Spaces end up being encoded in the url

Reply:
Space separated would be better as its the same format that is being used in other integrations too